Project.toml

```diff
@@ -10,6 +10,7 @@ Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
 Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
 DocStringExtensions = "ffbed154-4ef7-542d-bbb7-c09d3a79fcae"
 MLJTuning = "03970b2e-30c4-11ea-3135-d1576263f10f"
+Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
```

Comment:
This package has been used in RandomSeed module test but it seems that file hasn't been added into runtests.jl

Reply:
@yaxxie shall we add RandomSeed module  into the tests?

Reply:
No, we don't use seeds in the rest of the tests and we don't need the seed here, so please remove.

Reply:
RandomSeed.jl is using seeds but is not included in runtests.jl. Shall I remove RandomSeed.jl module ?

Reply:
We undertook an extensive effort to remove all uses of `seed!` and switch to property testing for running tests